sdk/jobs/automl-standalone-jobs/automl-image-classification-multiclass-task-fridge-items/automl-image-classification-multiclass-task-fridge-items.ipynb

```diff
@@ -378,6 +378,82 @@
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3.1. Using default hyperparameter values for the specified algorithm\n",
+    "Before doing a large sweep to search for the optimal models and hyperparameters, we recommend trying the default values for a given model to get a first baseline. Next, you can explore multiple hyperparameters for the same model before sweeping over multiple models and their parameters. This allows an iterative approach, as with multiple models and multiple hyperparameters for each (as we showcase in the next section), the search space grows exponentially, and  you need more iterations to find optimal configurations.\n",
+    "\n",
+    "If you wish to use the default hyperparameter values for a given algorithm (say `vitb16r224`), you can specify the job for your AutoML Image runs as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# general job parameters\n",
+    "compute_name = \"gpu-cluster\"\n",
+    "exp_name = \"dpv2-image-classification-experiment\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the AutoML job with the related factory-function.\n",
+    "\n",
+    "image_classification_job = automl.image_classification(\n",
+    "    compute=compute_name,\n",
+    "    # name=\"dpv2-image-classification-job-02\",\n",
+    "    experiment_name=exp_name,\n",
+    "    training_data=my_training_data_input,\n",
+    "    validation_data=my_validation_data_input,\n",
+    "    target_column_name=\"label\",\n",
+    "    primary_metric=\"accuracy\",\n",
+    "    tags={\"my_custom_tag\": \"My custom value\"},\n",
+    ")\n",
+    "\n",
+    "image_classification_job.set_limits(timeout_minutes=60)\n",
+    "\n",
+    "image_classification_job.set_image_model(model_name=\"vitb16r224\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Run the Command\n",
+    "Using the `MLClient` created earlier, we will now run this Command as a job in the workspace."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Submit the AutoML job\n",
+    "returned_job = ml_client.jobs.create_or_update(\n",
+    "    image_classification_job\n",
+    ")\n",
+    "\n",
+    "print(f\"Created job: {returned_job}\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ml_client.jobs.stream(returned_job.name)"
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {},
```

Comment:
I think we can remove the cell that defines compute_name and exp_name in this section since it is already defined above.